tests/test_co2eq_parameters.py

```diff
@@ -19,22 +19,23 @@ def get_possible_modes():
     with open("web/src/helpers/constants.js", encoding="utf-8") as file_:
```

Comment:
Forgot the get this one in
```suggestion
    with open("web/src/utils/constants.ts", encoding="utf-8") as file_:
```